pages/common/spatial.md

```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
+
+`spatial worker build --target=development --target=Osx`
+
+- Build workers for local deployment on Unreal:
+
+`spatial worker build --target=local --target=Windows`
+
+- Deploy locally:
+
+`spatial local launch {{launch_config}} --snapshot={{snapshot_file}}`
```

Comment:
So these commands operate on the project in the current directory like [npm](https://npmjs.org/)?

Reply:
Yes, that's correct